ironfish/src/rpc/adapters/tcpAdapter.test.ts

```diff
@@ -131,4 +134,30 @@ describe('TcpAdapter', () => {
       codeMessage: expect.stringContaining('this must be defined'),
     })
   })
+
+  it('should authenticate', async () => {
```

Comment:
It would be good to add a test that confirms that authentication matches when the tokens match

Reply:
yes we should be able to also make a request here to check that authentication does work too